norfair/drawing.py

```diff
@@ -19,6 +19,9 @@ def draw_points(
     radius: Optional[int] = None,
     thickness: Optional[int] = None,
     color: Optional[Tuple[int, int, int]] = None,
+    color_by_label: Optional[bool] = False,
+    draw_labels: Optional[bool] = False,
```

Comment:
These don't need to be optional, as optional would effectively let this variable have 3 states (true, false, None) but we only need 2.